packages/fixie/src/fixie-embed.tsx

```diff
@@ -16,25 +17,138 @@ export interface FixieEmbedProps extends React.IframeHTMLAttributes<HTMLIFrameEl
    */
   debug?: boolean;
 
+  /**
+   * If true, the iframe will be rendered in the DOM position where this component lives.
+   *
+   * If false, the iframe will be rendered floating on top of the content, with another iframe
+   * to be a launcher, à la Intercom.
+   */
+  inline?: boolean;
+
   /**
    * If you're not sure whether you need this, the answer is "no".
    */
   fixieHost?: string;
 }
 
+const defaultFixieHost = 'https://fixie.vercel.app';
+
 /**
  * A component to embed the Generic Fixie Chat UI on your page.
  *
  * Any extra props to this component are passed through to the `iframe`.
  */
-export function FixieEmbed({ speak, debug, agentId, fixieHost, ...iframeProps }: FixieEmbedProps) {
-  const embedUrl = new URL(`/embed/${agentId}`, fixieHost);
+export function InlineFixieEmbed({ speak, debug, agentId, fixieHost, ...iframeProps }: FixieEmbedProps) {
+  return <iframe {...getBaseIframeProps({ speak, debug, agentId, fixieHost })} {...iframeProps}></iframe>;
+}
+
+export function ControlledFloatingFixieEmbed({
```

Comment:
I might recommend docstrings for the various parameters here. I can add them later if needed.

Reply:
All the docstrings are already here. They come from `FixieEmbedProps`. If you hover over the param name in the editor, you can see the intellisense.

<img width="351" alt="image" src="https://github.com/fixie-ai/ai-jsx/assets/829827/234afae5-0ddf-4c4f-ae20-765c8d137785">